src/Components/Publishing/Display/DisplayAd.tsx

```diff
@@ -31,20 +33,28 @@ export const DisplayAd: SFC<DisplayAdProps> = props => {
   }
 
   const [width, height] = adDimension.split("x").map(a => parseInt(a))
+  const ad = (
+    <GPT
+      adUnitPath={`/21805539690/${props.adUnit}`}
+      targeting={props.targetingData}
+      slotSize={[width, height]}
+      onSlotRenderEnded={e => {
+        isAdEmpty = e.isEmpty
```

Comment:
Inside of an SFC this `isAdEmpty` will potentially get reset on each render cycle. If you'd like to keep some state around, best to use state:

```js
const [isAdEmpty, setAdEmpty] = useState(false)

...
onSlotRenderEnded={event => {
  setAdEmpty(event.isEmpty)
}
```